pkg/api/mlflow/common/helpers_test.go

```diff
@@ -0,0 +1,42 @@
+package common
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestGetContentType the TestGetContentType private helpers
```

Comment:
I think we don't need comments in tests since the name of test give the context.

Reply:
removing